### PR TITLE
added restaurant map

### PR DESCRIPTION
**Pull request by nhauke**
Added map of restaurant locations;
Added latitude/longitude to restaurants.hql, moved trunc_restaurants.hql to restaurants.hql;
Changed Makefile to reflect move of trunc_restaurants.hql

### CODE REVIEW

File: src/data/restaurants.hql [hunk 1]
```diff
@@ -4,10 +4,8 @@ SET hive.groupby.orderby.position.alias=true;
 
 ADD JAR /home/hadoop/Yelp/json-serde-1.3.7-jar.jar;
 
-DROP TABLE IF EXISTS restaurants;
```
Comment (jbrambleDC) [inline src/data/restaurants.hql:7]: Why did you remove the DROP TABLE IF EXISTS? @nhauke

**Comment by jbrambleDC** [comment]
LGTM 👍

File: src/data/restaurants.hql (hunk 1 above)
Comment (nhauke) [inline src/data/restaurants.hql:7]: Just below it I changed CREATE TABLE to CREATE TABLE IF NOT EXISTS
That way the table only has to be created once.
